src/validator.js

```diff
@@ -36,6 +37,7 @@ export default class Validator {
     this.chalk = new chalk.constructor(
       {enabled: !this.config.boring});
     this.collector = new Collector();
+    this.addonMetaData = {};
```

Comment:
I can integrate that into this PR no problem. Have to do a rebase anyway after #251 lands :smiley: